templates/wizard.php

```diff
@@ -59,7 +59,7 @@
 	and <a href="http://owncloud.org/promote"
 		target="_blank">spread the word</a>!')); ?>
 <?php else: ?>
-© 2011-<?php p(date('Y')) ?> <a href="https://owncloud.com" target="_blank">ownCloud Inc.</a>
```

Comment:
We don't know if `getEntity()` started in 2011. Plz compare with [defaults.php#L85](https://github.com/owncloud/enterprise/blob/master/theme/defaults.php#L85) in EE theme.